src/ripple/ledger/impl/PaymentSandbox.cpp

```diff
@@ -180,6 +213,19 @@ PaymentSandbox::balanceHook (AccountID const& account,
     return adjustedAmt;
 }
 
+std::uint32_t
+PaymentSandbox::ownerCountHook (AccountID const& account,
+    std::uint32_t count) const
+{
+    std::uint32_t result = count;
+    for (auto curSB = this; curSB; curSB = curSB->ps_)
+    {
+        if (auto adj = curSB->tab_.ownerCount (account))
+            result = std::max (result, *adj);
```

Comment:
This is another case where we always pick the largest saved `ownerCount`.  I think I see why you're making that choice (for one thing it's conservative).  But it would be useful to leave a comment with a justification.

I'm vaguely thinking of a case where offers are consumed on an account that will supply XRP liquidity in the path.  As those offers are consumed the ownerCount on the account will decrease and the available XRP will increase.  Using `max` will be conservative (you'll never chew into the reserve) but might not always supply the maximum possible XRP liquidity.